wgpu-core/src/binding_model.rs

```diff
@@ -122,6 +130,47 @@ pub struct BindGroup<B: hal::Backend> {
     pub(crate) life_guard: LifeGuard,
     pub(crate) used: TrackerSet,
     pub(crate) dynamic_count: usize,
```

Comment:
this should now be unnecessary, as one would just take `dynamic_binding_info.len()` instead